README.md

```diff
@@ -146,6 +147,10 @@
 
 - [vgram](https://github.com/dariotarantini/vgram) - Telegram bot library.
 
+### IRC
+
+- [vitric](https://github.com/m-242/vitric) - A transparent IRC library
```

Comment:
Please use a dot symbol at the end of the sentence.

Reply:
> ✖  152:44  List item description must end with proper punctuation  remark-lint:awesome-list-item
wow this is cool